lib/active_model/serializer.rb

```diff
@@ -28,8 +28,8 @@ def self.inherited(base)
       base._attributes_keys = self._attributes_keys.try(:dup) || {}
       base._associations = self._associations.try(:dup) || {}
       base._urls = []
-      serializer_file = File.open(caller.first[/^[^:]+/])
-      base._cache_digest = Digest::MD5.hexdigest(serializer_file.read)
+      serializer_file_path = caller.first[/\A\S+(?=:\d+:in)/]
+      base._cache_digest = Digest::MD5.hexdigest(File.read(serializer_file_path))
     end
```

Comment:
using the [appveyor](http://www.appveyor.com/) is right, since current testing case running in windows can already cover this problem.

But seems its complex and really necessory? Just relay on me because I will definitely working on windows and always report/fix issue if found. 

As Windows user, really don't expect we will be regarded as first class support tie like Mac/Linux in Rails ecosystem. :sob: